cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/home/lkloh/rapidnet/pybindgen/.waf-1.7.15-9c6c439a6416a92b3e844736c4ef3c7b/waflib/Tools/c_config.py", line 458, in run_c_code\n    bld.compile()\n  File "/home/lkloh/rapidnet/pybindgen/.waf-1.7.15-9c6c439a6416a92b3e844736c4ef3c7b/waflib/Build.py", line 188, in compile\n    raise Errors.BuildError(self.producer.error)\nBuildError: Build failed\n -> task in \'testprog\' failed (exit status 1): \n\t{task 45093456: cxx test.cc -> test.cc.1.o}\n[\'/usr/bin/g++\', \'-O2\', \'-g\', \'-Wall\', \'-std=c++11\', \'-DPYTHONDIR="/usr/local/lib/python2.7/dist-packages"\', \'-DPYTHONARCHDIR="/usr/local/lib/python2.7/dist-packages"\', \'-DHAVE_PYTHON_H=1\', \'-DHAVE_STDINT_H=1\', \'-DHAVE_BOOST_SHARED_PTR_HPP=1\', \'../test.cc\', \'-c\', \'-o\', \'test.cc.1.o\']\n'
